stats/buckets.go

```diff
@@ -76,6 +89,11 @@ type Buckets struct {
 
 var _ message.Message = &Buckets{}
 
+func (b *Buckets) String() string {
```

Comment:
Make it by value, so non-pointer `Buckets` values can be printed correctly in `fmt.Printf`.